src/dot_local/bin/executable_transcribe.py.tmpl

```diff
@@ -0,0 +1 @@
+{{ include "../bazel-bin/src/transcriber/transcribe.py" }}
```

Comment:
I feel like there is a better way for us to build a reusable chezmoi inclusion here to install/synchronize bazel built utilities but I'm not coming up with anything amazing on my own